src/explore-education-statistics-common/src/components/Banner.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from '@common/components/Banner.module.scss';
+import useMounted from '@common/hooks/useMounted';
+import ButtonText from '@common/components/ButtonText';
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  showBanner: boolean;
+  wide?: boolean;
+  onClose: () => void;
+}
+
+export default function Banner({
+  children,
+  showBanner,
+  wide = false,
+  onClose,
+}: Props) {
+  const { isMounted } = useMounted();
```

Comment:
Wouldn't do an `isMounted` check in here as this is a pretty basic component, so it should delegate it's rendering lifecycle to the parent component.

This is particularly relevant when it comes to SSR as this check is redundant in the admin.